src/Compilers/CSharp/Portable/BoundTree/InterpolatedStringBuilderData.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal sealed record InterpolatedStringBuilderData(
+        TypeSymbol BuilderType,
+        MethodArgumentInfo? Construction,
+        ImmutableArray<MethodArgumentInfo> BuilderFormatCalls,
+        MethodArgumentInfo? DisposeInfo,
+        uint ScopeOfContainingExpression);
```

Comment:
>ScopeOfContainingExpression [](start = 13, length = 27)

What exectly is this?

Reply:
I'll add a doc comment. It's the scope current expression during binding, for the purposes of setting the ValEscapeScope of the interpolated string builder.

---
In reply to: [610639085](https://github.com/dotnet/roslyn/pull/52422#discussion_r610639085) [](ancestors = 610639085)